dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/ConnectorArguments.java

```diff
@@ -157,11 +157,11 @@ public class ConnectorArguments extends DefaultArguments {
           .describedAs("com.company.Driver");
   private final OptionSpec<String> optionUri =
       parser
-          .accepts(OPT_URI, "JDBC driver URI (overrides host, port, etc if given)")
+          .accepts(OPT_URI, "JDBC driver URI")
```

Comment:
What is the reason for this change? The option is used by various connectors. Do you want to make this change for all of them?

Reply:
The overriding behavior isn't changed in this PR, but the removed description was incorrect. In practice each connector is expected to implement its own handling of this collision.

In most cases host/port/sid, etc. is ignored as promised, but this can't be guaranteed by ConnectorArguments.

Reply:
Right, it can't be guaranteed by `ConnectorArguments` class. Rather the help string is just a description of the behavior of the app (even if internally it is implemented in many places). The overriding behavior could potentially be refactored to a common class, so that the rule would be implemented in just one place.

Was there a connector that broke this promise?

Reply:
In general I think it's better to break this promise - when there's ambiguity because the user passed a redundant flag, failing is better than arbitrarily choosing one option.

Changing all old connectors would be too risky, but Oracle connectors are meant to fail if there's ambiguous configuration - this includes things like providing service name together with sid or providing the full url together with the host name.